editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -1430,6 +1535,7 @@
         repaint-timer (ui/->timer 30 "refresh-form-view"
                                   (fn [_timer _elapsed]
                                     (g/node-value view-id :form-view)))]
+    (g/node-value view-id :form-view)
```

Comment:
This requests cljfx to render the UI. Since `make-form-view` is called from UI thread, with new cljfx version we get immediate UI render instead of delaying it to `repaint-timer` tick.